packages/discord.js/src/structures/SelectMenuBuilder.js

```diff
@@ -21,18 +21,31 @@ class SelectMenuBuilder extends BuildersSelectMenu {
     );
   }
 
+  /**
+   * Normalizes a select menu option emoji
+   * @param {SelectMenuOptionData|JSONEncodable<APISelectMenuOption>} selectMenuOption The option to normalize
+   * @returns {Array<SelectMenuOptionBuilder | APISelectMenuOption>}
+   * @private
+   */
+  static normalizeEmoji(selectMenuOption) {
+    if (isJSONEncodable(selectMenuOption)) {
+      return selectMenuOption;
```

Comment:
The emoji is not being normalized here

Reply:
It doesn't need to be, for builder classes toJSON normalizes the emoji.